ui/narrative/methods/annotate_contigsets/spec.json

```diff
@@ -29,12 +29,65 @@
         "textarea_options" : {
             "n_rows" : 10
     	}
+  }, {
+    "id" : "scientific_name",
+    "optional" : true,
+    "advanced" : true,
+    "allow_multiple" : false,
+    "default_values" : [ "unknown taxon" ],
+    "field_type" : "text"
+  }, {
+    "id" : "domain",
+    "optional" : true,
+    "advanced" : true,
+    "allow_multiple" : false,
+    "default_values" : [ "U" ],
```

Comment:
Should U be the default? You seem to use B above (and it's not a dropdown option).

Reply:
Thanks for catching that. Fixed.
@JamesJeffryes